pica/ml/classifiers/svm.py

```diff
@@ -4,29 +4,103 @@
 #
 from time import time
 from typing import List, Tuple, Dict
-from collections import defaultdict
 
-import array
 import numpy as np
-import scipy.sparse as sp
 
-from sklearn.utils.fixes import sp_version
 from sklearn.pipeline import Pipeline
 from sklearn.svm import LinearSVC
 from sklearn.metrics import balanced_accuracy_score
 from sklearn.model_selection import StratifiedKFold, LeaveOneGroupOut
 from sklearn.feature_selection import RFECV
 from sklearn.calibration import CalibratedClassifierCV
-from sklearn.feature_extraction.text import CountVectorizer
 
+from pica.ml.vectorizer import CustomVectorizer
 from pica.structure.records import TrainingRecord, GenotypeRecord
 from pica.ml.cccv import CompleContaCV
+from pica.ml.trex_classifier import TrexClassifier
 from pica.util.logging import get_logger
 from pica.util.helpers import get_x_y_tn, get_groups
 from pica.ml.feature_select import recursive_feature_elimination, compress_vocabulary, DEFAULT_STEP_SIZE,\
     DEFAULT_SCORING_FUNCTION
 
 
+class TrexSVM(TrexClassifier):
+    def __init__(self, C: float = 5, penalty: str = "l2", tol: float = 1,
```

Comment:
If we annotate values as float, it'd be nice to initialize them as float as well,
e.g. `C: float = 5.` and `tol: float = 1.`
This applies to all classes here.

Reply:
Yeah, default values make sense, definitely.

Sorry to say I haven't looked into the test suite at all, but I surely can take some inspiration from `test_svm.py` to write analogous tests for `TrexClassifier`s.